src/foundry/common/abstract/backend.d.mts

```diff
@@ -192,14 +156,14 @@ declare abstract class DatabaseBackend {
   protected _logOperation(
     action: string,
     type: string,
-    documents: Document<any, any, any>[],
+    documents: Document.Any[],
     {
       parent,
       pack,
       level,
     }?: {
       /** A parent document */
-      parent?: Document<any, any, any>;
+      parent?: Document.Any;
```

Comment:
Consider adding explicit undefined to all of these object properties or just use `InexactPartial`.